src/libstd/time/mod.rs

```diff
@@ -95,8 +94,9 @@ impl Instant {
     ///
     /// # Panics
     ///
-    /// This function may panic if the current time is earlier than this instant
-    /// which can happen if an `Instant` is produced synthetically.
+    /// This function may panic if the current time is earlier than this
```

Comment:
Actually never mind me. It seems like the file is using 80 character-long lines, so this is fine. Sorry for the false alarm.